### PR TITLE
Dgmax null space polishing

### PR DESCRIPTION
**Pull request by Laar**
Polishing based on #54 that is not in the scope of that branch.

### CODE REVIEW

File: applications/DG-Max/DG-Max.cpp [hunk 1]
```diff
@@ -124,20 +124,31 @@ int main(int argc, char** argv) {
...
+        //                  << std::endl;
+        //        std::cout << "HCurl error " <<
+        //        errors[DGMaxDiscretizationBase::HCurl]
+        //                  << std::endl;
+        // Seems to be broken :(
+        //        DivDGMaxHarmonic<DIM> harmonicSolver(*mesh);
+        //        harmonicSolver.solve(harmonicProblem, divStab, p.getValue());
+        //        std::cout << "L2 error "
+        //                  << harmonicSolver.computeL2Error(harmonicProblem)
+        //                  << std::endl;
```
Comment (JensWehner) [inline applications/DG-Max/DG-Max.cpp:149]: pls remove

**Review by JensWehner — CHANGES_REQUESTED** [review]
Just remove commented out code. Otherwise nice to put the input functions into a map.

File: applications/DG-Max/DG-Max.cpp (hunk 1 above)
Comment (Laar) [reply applications/DG-Max/DG-Max.cpp:149]: I do like to keep it. This program is like the debugging/testing program. Thus I would like to be able to have a sample piece of code for each of the implementations that I can easily switch to.

A better solution may be possible, but is in my opinion not worth the time at the moment.

**Comment by JensWehner** [comment]
format is failing atm.

**Pull request merged**